components/suggest/src/yelp.rs

```diff
@@ -202,13 +205,13 @@ impl<'a> SuggestDao<'a> {
     /// Note that there should be only one record in `yelp_custom_details`
     /// as all the Yelp assets are stored in the attachment of a single record
     /// on Remote Settings. The following query will perform a table scan against
-    /// `yelp_custom_details` followed by an index search against `icons`, which
-    /// should be fine since there is only one record in the first table.
-    fn fetch_icon(&self) -> Result<Option<Vec<u8>>> {
-        Ok(self.conn.try_query_one(
+    /// `yelp_custom_details` followed by an index search against `icons`, `score`,
+    /// which should be fine since there is only one record in the first table.
```

Comment:
Could you update the first part of this comment please ("Fetch the icon for Yelp suggestions")? Also I don't think we need to add "score" to this sentence here. The sentence as written is still true and adding "score" makes it more confusing.